CHANGELOG.md

```diff
@@ -71,6 +71,7 @@ Inventory causes an error. Refs UIIN-2012.
 * Improve query when search by call number. Refs UIIN-2078.
 * Rename and reorder search options for clarity, consistency. Fixes UIIN-2060, UIIN-2061.
 * Browse contributors. Placeholder match does not look right in the contributor column. Refs UIIN-2079
+* Fix Cannot add tags to Instance when unlinked preceding/succeeding titles present. Fixes UIIN-2084.
```

Comment:
@mariia-aloshyna the ui-inventory has been already released. Please move this under `v9.2.0`:

https://github.com/folio-org/ui-inventory/blob/master/CHANGELOG.md#920-in-progress 

If this is intended for Morning Glory we will need to create a patch release.

Reply:
Hi @mkuklis fixed, thanks
Yes, it's Morning Glory and should be merged and released along with MODINV-674 as part of bugfix